docs/serving/using-auto-tls.md

```diff
@@ -0,0 +1,206 @@
+---
+title: "Using Auto-TLS in Knative"
+weight: 15
+type: "docs"
+---
+
+Within Knative, we provide a feature of automatically provisioning and 
+configuring TLS certificates to terminate the external TLS connection. This doc
+provides a guide about how to enable and configure Auto TLS feature in Knative.
+
+## Prerequisites
+
+<!-- TODO(zhiminx) add the link about installing Istio with SDS enabled after PR https://github.com/knative/docs/pull/1272 is checked in-->
+1. Follow the [instructions]() to install Istio with SDS enabled. Istio version
+needs to be 1.1 or above. And currently this feature does not work with 
+Gloo yet.
+2. Follow the [instructions](./installing-cert-manager.md) to install Cert-Manager.
+3. Follow the [instructions](../install) to install Knative cluster. Knative 
+version needs to be 0.6.0 or above.
+
+## Install Knative networking-certmanager components
+
+Skip this step if you already have Knative networking-certmanager components installed.
+Run below command to install Knative networking-certmanager components.
+```shell
+# KNATIVE_VERSION needs to be 0.6.0 or above.
+KNATIVE_VERSION=0.6.0
+
+kubectl apply --filename https://github.com/knative/serving/releases/download/v${KNATIVE_VERSION}/serving.yaml \ 
+-l networking.knative.dev/certificate-provider=cert-manager
+```
+
+## Configure Cert-Manager
+
+> Note that currently the Auto-TLS feature only supports DNS challenge.
+
+### Set up DNS challenge Provider
+
+Follow the [instructions](https://docs.cert-manager.io/en/latest/tasks/acme/configuring-dns01/index.html#supported-dns01-providers) to set up DNS challenge issuer according to the 
+DNS provider of your custom domain.
+
+#### Google Cloud DNS Provider Instruction
```

Comment:
I think this whole section should move out of these install instructions and stand alone as an example. Can we move all of this back into the "using-cert-manager-on-gcp.md" topic?

Reply:
I added the using-cert-manager-on-gcp.md back

Reply:
Thanks @ZhiminXiang ! Can you please add the "using-cert-manager-on-gcp.md" file to this PR, I have some edits to add. For example, I'd like to change the title to `Configuring cert-manager for Google Cloud DNS`